app/models/fact_check.rb

```diff
@@ -56,6 +57,11 @@ def self.get_exported_data(query, team)
     data
   end
 
+  def clean_tags
+    return if self.tags.blank?
+    self.tags = self.tags.map! { |tag| tag.strip.gsub(/^#/, '') }.uniq
```

Comment:
I think we should also handle case sensitive to avoid duplication.
i.e self.tags = self.tags.map { |tag| tag.strip.gsub(/^#/, '') }.map(&:downcase).uniq